index.js

```diff
@@ -2,18 +2,16 @@
 const Glue = require('@hapi/glue');
 const RestHapi = require('rest-hapi');
 const Manifest = require('./config/manifest.conf');
-
+const Sentry = require('@sentry/node');
+const Config = require('./config');
 const composeOptions = {
   relativeTo: __dirname,
 };
 
-const startServer = async function() {
+const startServer = async function () {
   try {
     const manifest = Manifest.get('/');
-    const server = await Glue.compose(
-      manifest,
-      composeOptions
-    );
+    const server = await Glue.compose(manifest, composeOptions);
```

Comment:
@franksmicro In general, it's best to keep formatting changes in a separate PR. Keeping them separate helps the reviewer focus on the important changes (in this case, Sentry related changes). Not a big deal here since there aren't a lot of changes, but a good thing to keep in mind.